solutions/standard/variables.tf

```diff
@@ -82,3 +90,41 @@ variable "cbr_rules" {
   default     = []
   # Validation happens in the rule module
 }
+
+########################################################################################################################
+# KMS variables
+########################################################################################################################
+
+variable "existing_kms_instance_crn" {
+  type        = string
+  default     = "crn:v1:bluemix:public:kms:us-south:a/abac0df06b644a9cabc6e44f55b3880e:d4559998-17ab-4ab8-ba9e-a0c9d2c4ef23::"
+  description = "The CRN of the key management service (KMS). If you are not using an existing KMS instance, you must specify this CRN. To support cross account functionality you must also provide a value for `ibmcloud_kms_api_key`."
```

Comment:
if I understand description correctly `If you are not using an existing KMS instance, you must specify this CRN.` it means I need to pass CRN to not use existing instance? is this correct?

Reply:
Modified.